project-base/src/Shopsys/ShopBundle/Resources/scripts/frontend/lazyLoadInit.js

```diff
@@ -0,0 +1,18 @@
+useNativeLazyload(new MiniLazyload({
+    rootMargin: '500px',
+    threshold: 0.5,
+    placeholder: 'placeholder.png'
+}, '.lazy'));
+
+(function ($) {
+
+    Shopsys = window.Shopsys || {};
+    Shopsys.lazyLoadCall = Shopsys.lazyLoadCall || {};
+
+    Shopsys.lazyLoadCall.inContainer = function (container) {
+        $(container).find('.lazyx').each(function () {
```

Comment:
Why is there a `x` in `find('.lazyx')`? Is that a typo or it has some reason?

Reply:
Oh yes, i forgot write comment. Btw I think so we should use selector like this `[loading=lazy]`. I waiting for TG reaction ...

Reply:
"x" was my mistake. I forgot test value. Removed 

Yes thats good point. Even in new minilazyload version is it recommended to use [loading=lazy] selector. Refactored. Please check it out.